TACC/Core-Portal
# Fixing migration & db export bugs

## Overview: ##
Running migrations shows these warnings,
```
WARNINGS:
 projects.AbstractProjectMetadata.co_pis: (fields.W340) null has no effect on ManyToManyField.
 projects.AbstractProjectMetadata.team_members: (fields.W340) null has no effect on ManyToManyField.
```
 dumpdata error,
```
rochaa$ docker exec core_portal_django python3 manage.py dumpdata >> ~/ws/core-data-migration/corev2-dump-after-v1-loaddata.json
CommandError: Unable to serialize database: 'CredentialsField' object has no attribute '_get_val_from_obj'

```
## Related Jira tickets: ##

* https://jira.tacc.utexas.edu/browse/FP-960

## Summary of Changes: ##
Removed unused model properties, generated missing migration, fixed to_dict error on export
## Testing Steps: ##
1. Follow steps here https://github.com/TACC/cep1to2/blob/main/README.md
2. Check that no warnings appear when running migrations
3. Run makemigrations and ensure no migrations are generated
4. Run a dumpdata using manage.py, it should complete successfully
## UI Photos:

## Notes: ##
